src/make-gecko-cron-hook.js

```diff
@@ -154,8 +154,11 @@ var makeHook = async function(projectName, project, options) {
           [
             'cd /builds/worker/checkouts/gecko',
             'ln -s /builds/worker/artifacts artifacts',
+            // Note:
+            //   --head-repository is actually the repoistory that gets passed into the decision taskk
+            //   --base-repository and --head-ref are ignored in post-esr60 branches
             './mach --log-no-times taskgraph cron --base-repository=$GECKO_BASE_REPOSITORY ' +
-            '--head-repository=$GECKO_HEAD_REPOSITORY ' +
+            `--head-repository=${project.repo} ` +
```

Comment:
I would like to continue using the env var for this, rather than repeating the repo name here.  At least, that's what we do for the previous command-line argument.

Reply:
`--base-repository` is ignored, so I don't bother changing it.

I could keep using an environment variable, but I'd have to make which variable it is conditional on whether this is for a comm-* repo. When I was first writing this would be the less complicated solution. But I am happy to switch back to using environment variables.

Reply:
Ah, ok, that makes sense.